### DISCUSSION TITLE
Is it possible to wire multiple output ports to one input node?

### DISCUSSION THREAD

**Discussion by hthomas92** [chrisjpatty/flume#214] [Q&A]
I couldn't connect multiple output ports to one input node. Is it possible to enable this?
When this is done, the input node could receive an array of objects.
This would also be useful if I use a graph to describe a control flow, or events and actions.

**Comment by wlievens** [comment]
I've had this same need. As a workaround I made a node that could "join" multiple inputs but it's clumsy.

**Comment by hthomas92** [comment]
Unfortunately that doesn't really work for me, the graphs would look too ugly.

**Reply by wlievens** [reply]
I agree, it's a clutch
